### PR TITLE
Kate Tester

### PR DESCRIPTION
**Pull request by katemorris**

### CODE REVIEW

**Comment by katemorris** [comment]
Things I have learned by looking at others pull requests, I need to commit more.

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  def is_full_time?
+    @hours_in_school_day > 4
+  end
+
+  def standard_student_names
+    formal_names = @student_names.map do |name|
+      name.capitalize
+    end
+    return formal_names
```
Comment (megstang) [inline lib/school.rb:26]: since map returns the altered array, no need to save this into a variable and return it.

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  end
+
+  def convert_end_time_to_clock_time
+    (self.end_time.to_i - 12).to_s + ":00"
```
Comment (megstang) [inline lib/school.rb:30]: What if you had a school that started at 9 and was 2 hours long? Or what if you had a school that started at 9 and was 3 hours long? If the end time is 12 or less, it looks like you may run into some issues with the way this extension is done. That said, way to take a shot at it!

File: lib/school.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def end_time
+    (@start_time.to_i + @hours_in_school_day).to_s + ":00"
+  end
+
+  def is_full_time?
+    @hours_in_school_day > 4
```
Comment (megstang) [inline lib/school.rb:19]: I like that you did this in just one line rather than doing a whole if statement. 👍 great thinking

**Pull request closed without merging**